core/models.py

```diff
@@ -1207,12 +1204,18 @@ def __unicode__(self):
         return "path=%s size=%s sha1=%s" % (self.path, self.size, self.sha1)
 
     def _add_page(self, page, tar):
+        from index import get_page_text
+
         d = page.issue.date_issued
         relative_dir = "%s/%i/%02i/%02i/ed-%i/seq-%i/" % (page.issue.title_id, d.year, d.month, d.day, page.issue.edition, page.sequence)
 
         # add ocr text
         txt_filename = relative_dir + "ocr.txt"
-        ocr_text = page.ocr.text.encode('utf-8')
+        ocr_text = get_page_text(page)
+        if len(ocr_text) > 0:
+            ocr_text = ocr_text[0].encode('utf-8')
+        else:
+            ocr_text = ['Text not available']
```

Comment:
This duplicates the constant returned by `get_page_text`. It isn't immediately obvious that this code path can even be called because if I'm reading https://github.com/lsukharn/chronam/blob/267daf1fb713353d70fd7a52043bd4ce407a2515/core/index.py#L669-L680 correctly `get_page_text` will always return a non-empty list.

Reply:
That's what I commented before, so should I keep this code or return to the previous implementation? (I wrote get_page_text with an idea that it will always returns non-empty list)

Reply:
yes, revert to what you had before. I wasn't following as closely as I should have, and as it was pointed it out, it will never be None

Reply:
Reverted to ocr_text = get_page_text(page)[0].encode('utf-8')